cinema/permissions.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework.permissions import BasePermission, SAFE_METHODS
```

Comment:
There's a syntax error in the import statement. It should be `from rest_framework.permissions import BasePermission, SAFE_METHODS`.